cmd/src/actions_exec.go

```diff
@@ -80,6 +81,10 @@ Examples:
 
 	$ src actions exec -f ~/run-gofmt.json | src campaign patchset create-from-patches
 
+  Execute an action and save the patches it produced to 'patches.json'
+
+	$ src actions exec -f ~/run-gofmt.json -o patches.json 
+
```

Comment:
Can you add a comment to the first example in this description, without `-create-patchset` & without a pipe, that the patches are saved by default into `patches.json`?